### DISCUSSION TITLE
Updating on the lifecycle

### DISCUSSION THREAD

**Discussion by akametall** [strimzi/strimzi-kafka-operator#5695] [Q&A]
Hello, can anybody help me with the question:
i want to update some reources like ram without rollout restart of the deployment. Can strimzi provide it? Cannot find answer in docs, but maybe it's i'm scattered.

**Comment by scholzj** [comment]
WDYM with _update some resources like ram_? As in change the amounts RAM allocated to the different pods? I think that always requires restart of the pods which are affected. That said, it should have minimal impact on the clients if you use replication etc.

**Reply by akametall** [reply]
I see the project for the rabbitmq with such features. and try to understand can there be something similar for kafka.
update some resources - i mean requests and limits, my bad not to write it clean.

**Reply by scholzj** [reply]
I do not think Kubernetes supports it unless something changed very recently.
